packages/websocket-plugin/src/websocket-handler.ts

```diff
@@ -37,7 +37,10 @@ export class WebSocketHandler {
         }
         store.dispatch({ ...msg, type });
       },
-      err => store.dispatch(new WebsocketMessageError(err)),
+      err =>
+        err instanceof CloseEvent
```

Comment:
Where does `CloseEvent` come from? I don't see any imports. It's it a browser type?

Reply:
@markwhitfeld https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent

Reply:
as @splincode  pointed, this is browser's websocket api, no import needed